cve.py

```diff
@@ -5,16 +5,23 @@ def __init__(self, dict : {}):
         """
 
         """
-        self.ID = dict['cve']['CVE_data_meta']['ID']
+        self.identifier = dict['cve']['CVE_data_meta']['ID']
         self.ASSIGNER = dict['cve']['CVE_data_meta']['ASSIGNER']
         self.description = dict['cve']['problemtype']['problemtype_data'][0]['description'][0]['value']
+        self.severity = dict['impact']
+        if len(self.severity) > 1:
+            self.severity = self.severity['baseMetricV3']['cvssV3']['baseSeverity']
+        else:
+            self.severity = self.severity['baseMetricV2']['severity']
+
+
 
     def getCveId(self) -> int:
         """
```

Comment:
What's the purpose of the comments in this file?

Reply:
remove this comments